client/my-sites/themes/theme-options.js

```diff
@@ -13,7 +13,8 @@ import config from 'config';
 import {
 	activateTheme,
 	installAndActivate,
-	installAndTryAndCustomize
+	installAndTryAndCustomize,
+	deleteTheme as deleteThemeAction,
```

Comment:
Fine to call it `deleteTheme` -- it's consistent with `activateTheme` a couple of lines above.